torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -119,8 +119,12 @@ void XlaUpdateTensors(
     tensorflow::gtl::ArraySlice<const at::Tensor> source_cpu_tensors,
     tensorflow::gtl::ArraySlice<const size_t> indices) {
   for (auto index : indices) {
-    XLATensor xtensor = GetXlaTensorUnwrap(dest_xla_tensors.at(index));
-    xtensor.UpdateFromTensor(source_cpu_tensors.at(index));
+    if (dest_xla_tensors.at(index).device().is_cpu()) {
```

Comment:
In any case, since the damage of multiple dispatch has already been done, above, use `TryGetXlaTensor()` instead of specifically checking for is-CPU.

Reply:
Argh!
Above does the same.
Here is one like: https://github.com/pytorch/xla/pull/1092
Either approve that, or do the same changes in yours and kill the above.
We also had that "unwrap business" still as empty shim.